dbt_platform_helper/commands/environment.py

```diff
@@ -399,18 +398,35 @@ def _generate_terraform_environment_manifests(
 ):
     env_template = setup_templates().get_template("environments/main.tf")
 
+    modules_version = _determine_terraform_platform_modules_version(
+        env_config, terraform_platform_modules_version
+    )
```

Comment:
`modules_version` feels like it could be any kind of modules, I would prefer it to be named for what it is.

I can see how you got there though. Maybe this approach would be acceptable?

```suggestion
    terraform_platform_modules_version = _determine_terraform_platform_modules_version(
        env_config, terraform_platform_modules_version
    )
```

Reply:
Renamed a load of these variables. Not a fan of reassigning variables as it can cause potential bugs, but hopefully this is OK.

Reply:
I would agree normally, but in this case it felt better than any alternative I could come up with that didn't involve mile long variable names, and I'm quite comfortable with a long variable name.